backend/src/session/Session.ts

```diff
@@ -124,6 +147,25 @@ export default class Session {
         this.response.setHeader(Session.SESSION_HEADER, '-');
     }
 
+    async destroyAll() {
```

Comment:
nit `Session#destroyAll` sounds like it will wipe all the session for all logged users
```suggestion
    async destroyAll4User() {
```